integration-tests/emulated/src/mock/network.rs

```diff
@@ -14,28 +14,32 @@
 // You should have received a copy of the GNU General Public License
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
+use asset_hub_rococo_emulated_chain::AssetHubRococo as AssetHubParachain;
+use rococo_emulated_chain::Rococo as RococoChain;
 use xcm_emulator::decl_test_networks;
 
-use crate::mock::{
-	para_chains::{AssetHubPolkadot, AssetHubRococo, Peregrine, Spiritnet},
-	relay_chains::{Polkadot, Rococo},
-};
+use crate::mock::para_chains::{PeregrineParachain, SpiritnetParachain};
+
+#[cfg(not(feature = "runtime-benchmarks"))]
+#[cfg(test)]
+pub mod chains {
+
+	pub use super::*;
+
+	pub type Rococo = RococoChain<MockNetwork>;
+	pub type Spiritnet = SpiritnetParachain<MockNetwork>;
+	pub type Peregrine = PeregrineParachain<MockNetwork>;
+	pub type AssetHub = AssetHubParachain<MockNetwork>;
+}
 
 decl_test_networks! {
```

Comment:
If this is used inside the `chains` module, why not declare it in there directly?

Reply:
I think at this point the `chains` module has little reason to exist.

Reply:
The types in chains are only used for the tests. Therefore, I got the warning of unused variables. I feature guided the mock and the utils too: [8470fc8](https://github.com/KILTprotocol/kilt-node/pull/655/commits/8470fc86d066f2229ff2082932afdfa88a616781)